card/src/test/java/com/adyen/checkout/card/AddressFormUtilsTest.kt

```diff
@@ -127,8 +127,8 @@ internal class AddressFormUtilsTest {
     }
 
     @Test
-    fun initializeCountryOptions_AddressConfigurationIsNone_ExpectEmptyList() {
-        val addressConfiguration = AddressConfiguration.None
+    fun `When Country Options are initialized that a Address Params is null ,Expect Empty List`() {
```

Comment:
Tests should start with lower case as it's the convention throughout the whole project.